src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -1109,6 +1111,126 @@ await using (var context = InMemoryApplicationDbContext(contextId))
+
+                var result = await releaseService
+                    .CreateReleaseStatus(
+                        release.Id,
+                        new ReleaseStatusCreateViewModel
+                        {
+                            ApprovalStatus = ReleaseApprovalStatus.Approved,
+                            LatestInternalReleaseNote = "Test note",
+                            PublishMethod = PublishMethod.Scheduled,
+                            PublishScheduled = "2051-06-30",
+                            NextReleaseDate = new PartialDate {Month="12", Year="2000"}
+                        }
+                    );
+
+                result.AssertRight();
+            }
+
+            MockUtils.VerifyAllMocks(releaseChecklistService, contentService, releaseFileService, preReleaseUserService);
+        }
+
+        [Fact]
+        public async Task CreateReleaseStatus_Draft_DoNotSendPreReleaseEmails()
+        {
+            var release = new Release
+            {
+                Type = new ReleaseType {Title = "Ad Hoc"},
+                Publication = new Publication {Title = "Old publication"},
+                ReleaseName = "2030",
+                Slug = "2030",
+                Version = 0,
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                await context.AddAsync(release);
+                await context.SaveChangesAsync();
+            }
+
+            var contentService = new Mock<IContentService>(MockBehavior.Strict);
+            var preReleaseUserService = new Mock<IPreReleaseUserService>(MockBehavior.Strict);
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                contentService.Setup(mock =>
+                        mock.GetContentBlocks<HtmlBlock>(release.Id))
+                    .ReturnsAsync(new List<HtmlBlock>());
+
+                preReleaseUserService.Setup(mock =>
+                        mock.SendPreReleaseUserInviteEmails(It.Is<Release>(r => r.Id == release.Id)))
+                    .ReturnsAsync(Unit.Instance);
+
+            var releaseService = BuildReleaseService(contentDbContext: context,
+                    contentService: contentService.Object,
+                    preReleaseUserService: preReleaseUserService.Object);
+
+                var result = await releaseService
+                    .CreateReleaseStatus(
+                        release.Id,
+                        new ReleaseStatusCreateViewModel
+                        {
+                            ApprovalStatus = ReleaseApprovalStatus.Draft,
+                            LatestInternalReleaseNote = "Test note",
+                        }
+                    );
+
+                result.AssertRight();
+            }
+
+            preReleaseUserService.VerifyNoOtherCalls();
```

Comment:
Instead of:

```
preReleaseUserService.VerifyNoOtherCalls();
MockUtils.VerifyAllMocks(contentService);
```

you can use:

```
MockUtils.VerifyAllMocks(contentService, preReleaseUserService);
```

This does the VerifyNoOtherCalls() for you.

Reply:
For some reason this broke the unit test - it seems VerifyAllMocks expects any Setup defined to get called. But since the mock isn't being used at all, I'll delete it. 

I wanted to define it to catch if it was getting called when it shouldn't - but I don't think that's right?